portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,30 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches stats from the servers and adds them to the DOM.
+ */
+function getComments() {
+  fetch('/comments').then(response => response.json()).then((comments) => {
+    // comments is an ArrayList, so we have to
+    // reference its fields by index to create HTML content
+
+    console.log(comments);
+
+    const commentsListElement = document.getElementById('comments-container');
+    commentsListElement.innerHTML = '';
+    for (let i = 0; i < comments.length; i++) {
+      commentsListElement.appendChild(
+        createListElement('Comment ' + (i + 1) + ': ' + comments[i])
```

Comment:
Oops forgot to mention this too. I think you can use template literals to make the string easier to read. e.g 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals